### ISSUE TITLE
JSAssert annotation

### ISSUE THREAD

**Issue by zsalab** [nomemory/java-bean-validation-extension#5]
I am happy to find this project, I really appreciate the effort you put in it. I would like to propose to detach the JSAssert validation functionality to a separate module because bring the heavy graalvm JS engine which is itself 15M+
I think would be a great idea to separate that functionality to separated jar

**Comment by nomemory** [comment]
@zsalab You are right. I will do that when I will have some available time.

**Comment by Tosken1337** [comment]
GraalVM js engine is still in the compile time dependencies. I think this could be removed now?

**Comment by nomemory** [comment]
Yes, it can be removed. I will try to make a new release without it.
